src/com/nutrons/steamworks/Shooter.java

```diff
@@ -41,29 +43,46 @@ public Shooter(LoopSpeedController shooterController, Flowable<Boolean> shooterB
     this.shooterButton = shooterButton;
     this.distance = distance;
     this.setpointHint = setpointHint;
-    this.variableSetpoint = this.distance.filter(x -> x != 0.0).map(x -> 11.778 * x + 1858.7).share();
+    this.variableSetpoint = this.distance.filter(x -> x != 0.0).map(x -> 11.778 * x + 1808.7).share();
+  }
+
+  public Command auto() {
+    Flowable<ControllerEvent> setpoint = Flowable.just(AUTO_SETPOINT).map(aimEvent);
+    return Command.fromSubscription(() ->
+        setpoint.subscribe(shooterController))
+        .addFinalTerminator(() -> shooterController.accept(stopEvent));
   }
 
   public Command pulse() {
     Flowable<ControllerEvent> combined = setpointHint.withLatestFrom(Flowable.just(SETPOINT)
-        .mergeWith(variableSetpoint.take(1)), (x, y) -> x + y).map(aimEvent);
+        .mergeWith(
```

Comment:
Whats going on here? We use pulse in teleop. Now, we are getting the values from the SD instead of the variable setpoint. I'm not sure shooter setpoint will work in teleop properly, unless I'm missing something.